packages/live-preview-sdk/src/saveEvent.ts

```diff
@@ -42,10 +52,15 @@ export class SaveEvent {
   ): void {
     if (message.method === LivePreviewPostMessageMethods.ENTRY_SAVED && this.subscription) {
       const { entity } = message as EntrySavedMessage;
-      const entries = getAllTaggedEntries({ options: this.options });
+      if (this.inspectorModeEnabled) {
+        const entries = getAllTaggedEntries({ options: this.options });
 
-      if (entries.includes(entity.sys.id)) {
+        if (entries.includes(entity.sys.id)) {
+          this.subscription(entity);
+        }
+      } else {
         this.subscription(entity);
+        window.location.reload();
```

Comment:
Note: Do we want to hardcode this or have users provide it in the callback if they wish for reloading?

Reply:
user have to provide it - we should have an example in the examples directory for it

Reply:
the reason for providing it, you could have a reload implementation by the framework you're using for routing (e.g. next.js, react-router, ..)

Reply:
Makes sense, I'll remove this line then!